persistence-modules/hibernate5/src/main/resources/log4j.properties

```diff
@@ -1 +1,6 @@
 log4j.rootLogger=INFO, stdout
+
```

Comment:
We can revert these changes since hibernate5 module does not produce a lot of logs. Consequently, below changes may be out of scope

Reply:
@anastasiosioannidis If we revert this change the test BootstrapAPIIntegrationTest will output the following  `log4j:ERROR Could not find value for key log4j.appender.stdout
log4j:ERROR Could not instantiate appender named "stdout".
log4j:WARN No appenders could be found for logger (org.hibernate.Version).
log4j:WARN Please initialize the log4j system properly.
`  I will go ahead and revert it, be i just need to mention the reason why it was added.